ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -1,5 +1,7 @@
 # Defaults file for the compatibility layer role.
 ---
+eessi_overlay_version: "2020.10"
```

Comment:
We should bump this to `2020.11` (we won't be rebuilding the `2020.10` compat layer anyway)?

Reply:
Done. Also changed the name of that variable to just `eessi_version`, as overlay means something else in this context.